manager/controllers/system_detail.go

```diff
@@ -34,12 +34,9 @@ func SystemDetailHandler(c *gin.Context) {
 	var systemItemAttributes SystemItemAttributes
 	query := database.Systems(database.Db, account).
 		Select(database.MustGetSelect(&systemItemAttributes)).
+		Joins("JOIN inventory.hosts ih ON ih.id = inventory_id").
 		Where("sp.inventory_id = ?::uuid", inventoryID)
 
-	if applyInventoryHosts {
```

Comment:
Please, set applyInventoryHosts to default value false to use the same value in tests and in prod.